src/hardware/setup.rs

```diff
@@ -275,7 +275,7 @@ pub fn setup(
         .use_hse(8.MHz())
         .sysclk(design_parameters::SYSCLK.convert())
         .hclk(200.MHz())
-        .per_ck(design_parameters::TIMER_FREQUENCY.convert())
+        .per_ck(64.MHz()) // 64 MHz HSI clk selected per default (this setting changes nothing)
```

Comment:
```suggestion
        .per_ck(64.MHz()) // HSI, only used for internal ADC. This is not the "peripheral" clock for timers and others.
```

Reply:
Technically speaking, the `per_ck` _can_ be the peripheral "kernel" clock for some peripherals (i.e. SPI), but we don't use it as that: 
![image](https://user-images.githubusercontent.com/8771450/203294016-e8b4d995-3afd-4197-9881-8c688636233b.png)

Reply:
Yes. As the comment says, it is only used for ADC and not for anything else.